packages/apple-pay-integration/src/mocks/apple-pay-button.mock.ts

```diff
@@ -3,13 +3,316 @@ import { CheckoutButtonInitializeOptions } from '@bigcommerce/checkout-sdk/payme
 import { WithApplePayButtonInitializeOptions } from '../apple-pay-button-initialize-options';
 import ApplePayButtonMethodType from '../apple-pay-button-method-type';
 
+export interface CustomItem {
```

Comment:
I think almost all interfaces added here can be taken from `payment-integrations-test-utils` package

Reply:
removed